Dockerfile.API

```diff
@@ -4,19 +4,21 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS installer-env
 # https://www.reddit.com/r/docker/comments/i34thd/dotnet_core_slow_restore/g0azda1?utm_source=share&utm_medium=web2x&context=3
 # It does help, we went from 3m20s to build after changes to 1m20s
 
-COPY src/API/*.csproj ./src/API/
-COPY src/Database/*.csproj ./src/Database/
-COPY src/Models/*.csproj ./src/Models/
+COPY src/API/*.csproj ./app/src/API/
+COPY src/Database/*.csproj ./app/src/Database/
+COPY src/Models/*.csproj ./app/src/Models/
+WORKDIR  /app
```

Comment:
Just for posterity the changes to the `COPY` and addition of `WORKDIR` came from a handy [GitHub issue](https://github.com/dotnet/dotnet-docker/issues/644#issuecomment-423051438).